packages/ui-kit/src/Panel/index.tsx

```diff
@@ -22,9 +22,11 @@ export function Panel({
   onClose,
   customCloseBtn,
   classes,
+  ref,
```

Comment:
hm, it's not used in Panel component
it looks like you were going to attach it to some element 🤔

Reply:
it was a kind of quickfix: **React.DetailedHTMLProps** has **ref** prop inside that has **LegasyRef** type while **Card** element uses ForwardRef and requires ref  of type **RefObject**. These two types are not compatible, so, I decided just to get rid of ref prop in props that are passed from Panel to Card and this is why I added ref here. Will think about more robust solution for this problem